# publishes robot ID
# publishes if robot needs to halt completely or only stops moving until a strategy found
# specifies duration of halt

string RobotId
bool halt
float32 time_to_halt